### ISSUE TITLE
Connection and integration button alignment

### ISSUE THREAD

**Issue by squakez** [syndesisio/syndesis#8047]
## This is a...

<pre><code>
[ ] Feature request
[x] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description

Apparently connection and integration action buttons are no longer aligned on the right as done through the rest of the application. We should either keep all buttons aligned on the right or on the left.

### Left hand buttons

![Screenshot_2020-03-06 Connections - Syndesis](https://user-images.githubusercontent.com/23169723/76089178-800ac780-5fb9-11ea-89af-ce9c86e5d3ec.png)

![Screenshot_2020-03-06 Integrations - Syndesis](https://user-images.githubusercontent.com/23169723/76089198-87ca6c00-5fb9-11ea-96c1-472d1a9a8129.png)

### Right hand buttons

![Screenshot_2020-03-06 Save or Add Step - Syndesis(1)](https://user-images.githubusercontent.com/23169723/76089343-c19b7280-5fb9-11ea-9271-e197a56d1cb6.png)

I have seen this in connection, integration and api provider creation.

**Comment by gashcrumb** [comment]
Yep, the first two are due to the pf4 migration, it appears the pf4 toolbar doesn't have a way to right-justify the buttons in the way the old pf3 toolbar did.

@dongniwang FYI

**Comment by dongniwang** [comment]
@gashcrumb @squakez thank you both! I'll add this to the PF4 migration doc. We'll have a visual pass before finalizing everything.
